Sources/Build/BuildPlan.swift

```diff
@@ -544,6 +562,8 @@ public final class SwiftTargetBuildDescription {
         args += moduleCacheArgs
         args += buildParameters.sanitizers.compileSwiftFlags()
         args += ["-parseable-output"]
+        args += ["-F", buildParameters.buildPath.pathString]
+        args += ["-L", buildParameters.buildPath.pathString]
```

Comment:
This shouldn't be required since we don't link here.

Reply:
I can't remember why I added it, but it does seem un-necessary indeed.